django/contrib/gis/gdal/raster/const.py

```diff
@@ -43,3 +43,13 @@
     'Average': 5,
     'Mode': 6,
 }
+
+# Fixed basepath for buffer based GDAL in memory files.
+VSI_FILESYSTEM_BASEPATH = '/vsimem/'
+
+# Flag for managing ownership over buffer when creating VSI in
+# memory files.
+VSI_TAKE_BUFFER_OWNERSHIP = False
```

Comment:
Are these boolean constants just for readability or are these settings meant to be monkeypatched for alternate behavior?